src/routes/+page.svelte

```diff
@@ -99,6 +99,34 @@
   </Row>
 </Section>
 
+<Section id="sponsorships" color="var(--blue)" padding="60px">
+  <Row gap={84} reverse
+    ><figure>
+      <img
+        src={data.info.homepageSponsorships.src}
+        alt={data.info.homepageSponsorships.alt}
+      />
+    </figure>
+    <div id="sponsorship-content">
+      <h2>
+        We
+        <Icon icon="heart" width="1em" height="1em" inline={true} />
+        Our Sponsors
+      </h2>
+      <div class="row-center" />
+      <p>
+        Our drive to write impactful software for social good is made possible
```

Comment:
Could we reduce this text to 2-3 lines when rendered? It's a little verbose.

Reply:
We could just delete the first sentence honestly.